app/javascript/src/components/Dashboard/Notes/List.jsx

```diff
@@ -0,0 +1,57 @@
+// eslint-disable-next-line react/jsx-filename-extension
```

Comment:
Why is this required here? Remove these comments which are disabling eslint rules. Unless your consult with the tech lead you should not be adding these comments.

Reply:
Removed it.